orbit-2-coroutines/README.md

```diff
@@ -0,0 +1,55 @@
+# Orbit 2 Coroutines plugin
+
+The coroutine plugin provides:
+
+- Coroutine DSL operators
```

Comment:
```suggestion
- Coroutine operators
```

Reply:
tempted to suggest merging this with the operators, as currently this sentence doesn't add much value.

Then:

> The coroutine plugin provides the following operators:
> - transformSuspend
> - transformFlow